src/bio/bio_xstream.c

```diff
@@ -394,6 +394,12 @@ bio_nvme_init(const char *storage_path, const char *nvme_conf, int shm_id,
 		goto free_mutex;
 	}
 
+	if (strlen(nvme_conf) == 0) {
+		D_INFO("NVMe config isn't specified, skip NVMe setup.\n");
+		nvme_glb.bd_nvme_conf = NULL;
+		return 0;
+	}
+
 	fd = open(nvme_conf, O_RDONLY, 0600);
 	if (fd < 0) {
 		D_WARN("Open %s failed("DF_RC"), skip DAOS NVMe setup.\n",
```

Comment:
Is this now an error?  If a file is specified but it doesn't exist then that should be more serious than a default file not existing.

Reply:
For VOS standalone mode (daos_perf VOS mode), it'll just use the default config (/etc/daos_nvme.conf) if it's present, so if we change it to error here, it'll fail VOS standalone mode when the /etc/daos_nvme.conf isn't present.